polyfill/lib/instant.mjs

```diff
@@ -196,3 +196,13 @@ MakeIntrinsicClass(Instant, 'Temporal.Instant');
 function bigIntIfAvailable(wrapper) {
   return typeof BigInt === 'undefined' ? wrapper : wrapper.value;
 }
+
+function bigIntFloorDiv(left, right) {
```

Comment:
Do we repeat functions like this elsewhere? Would it be worth turning this function into a shared implemention in one file?